code/modules/shuttles/shuttles_web.dm

```diff
@@ -1,12 +1,11 @@
 //This shuttle traverses a "web" of route_datums to have a wider range of places to go and make flying feel like movement is actually occuring.
-/datum/shuttle/web_shuttle
+/datum/shuttle/autodock/web_shuttle
 	flags = SHUTTLE_FLAGS_NONE
```

Comment:
IMO should have the same flags as the others. Why not? Maybe not process, but the ZERO_G one at least.